functions/src/sgyfetch.ts

```diff
@@ -131,14 +133,14 @@ export const fetchMaterials = functions.https.onCall(async (data, context) => {
 
         // Fetch courses, then kick out yucky ones
         // TODO: type this better
-        const courses_unfiltered = await get(`users/${sgyInfo.uid}/sections`, sgyInfo.key, sgyInfo.sec);
-        const gunn_student_course = courses_unfiltered.section.find((sec: {section_title: string}) => {
+        const unfiltered = await get(`users/${sgyInfo.uid}/sections`, sgyInfo.key, sgyInfo.sec);
+        const gunnStudentCourse = unfiltered.section.find((sec: {section_title: string}) => {
             const title = sec.section_title.toLowerCase();
             return (title.endsWith(YEAR_STR) && title !== YEAR_STR);
         })
-        const grad_year = ['se', 'ju', 'so', 'fr'].indexOf(gunn_student_course.section_title.slice(0,2)) + CURRENT_YEAR + 1;
+        const grad_year = ['se', 'ju', 'so', 'fr'].indexOf(gunnStudentCourse.section_title.slice(0,2)) + CURRENT_YEAR + 1;
```

Comment:
Is there a better way to get graduation year via the schoology API? I recall when digging into obscure properties in people's `userData` objects on firestore that some people had a `gradYear` lumped into their schoology data. Is there a way we could fetch this instead of the gunn student course?

Reply:
We could, but it would require a separate fetch to /users/me, and iirc the gradYear is also off by one year for no reason at all. Might just be better to just parse courses instead.